apps/hash-ai-worker-ts/src/activities/infer-entities/infer-entity-summaries.ts

```diff
@@ -276,7 +284,7 @@ export const inferEntitySummaries = async (params: {
           content: dedent(`
                    You did not suggest any entities of the following entity types: ${typesWithNoSuggestionsToRerequest.join(
                      ", ",
-                   )}. Please reconsider the input text to see if you can identify any entities of those types.
+                   )}. Please reconsider the input text${existingEntities && existingEntities.length > 0 ? " and the existing entities" : ""} to see if you can identify any entities of those types.
```

Comment:
> Please reconsider the input text and the existing entities to see if you can identify any entities of those types.

What do we want it to do here? 'of those types' is referring to the 'types with no suggestions' list... is it something like this?

> Please reconsider the input text to see if you can identify any entities of those types, considering whether any links can be created to existing entities also.

Could potentially drop the reference to 'existing entities' if none of the outstanding types are links.

Reply:
I've reworked this in https://github.com/hashintel/hash/pull/4346/commits/ae9ae0a5d000e20bdc5732063f47fc05ec7d853f to be more specific about what is missing, and have dropped the reminder about existing entities if no link types are missing.